docs/release-notes/8.1.asciidoc

```diff
@@ -47,6 +47,7 @@
 ==== Known issues
 * An {endpoint-sec} integration bug prevents benign Windows files from being deleted under certain circumstances.
 * On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
+* Avoid using indices populated from value lists for indicator match rules. Indicator match rules cannot use the `.lists` system index and will encounter execution errors when ran ({pull}133457[#133457]).
```

Comment:
```suggestion
* Indicator match rules cannot use the `.lists` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules. ({pull}133457[#133457]).
```
I think switching the order of the sentences would make this more consistent with the other known issues listed above it, since they state the problem before telling people what to do about it. If you agree, the same applies below.

Reply:
Ah good catch. I'll make the change in the other area as well - thanks!